src/plays/drawing-board/modules/strokes/slice.ts

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { RootState } from "../../utils/types";
+import { endStroke, SharedAction } from "../sharedActions";
+
+const initialState: RootState["strokes"] = [];
+
+const strokes = createSlice({
+  name: "strokes",
+  initialState,
+  reducers: {},
+  extraReducers: (builder) => {
+    builder.addCase(endStroke, (state, action) => {
```

Comment:
make sure all has use cases coz the same reducer with `endStroke` is created in folder `currentStroke` -> `slice.ts` and historyStroke -> slice.ts

Reply:
In the `currentStroke`->`slice.ts` we are using `endStroke` in `extraReducers` to reset the currentStrokes points array.

In the `historyStroke`->`slice.ts`, `endStoke` is used because we also need handle the `END_STROKE` action to reset the `historyIndex` to 0.